src/mmrm.cpp

```diff
@@ -151,8 +149,6 @@ Type objective_function<Type>::operator() ()
   REPORT(beta_vcov);
```

Comment:
in line 115 we already do the decomposition of the beta_cov. that should be sufficient so that we don't have to do it again - if we report it back to the R side?

Reply:
yes no decompose is needed in prediction, only the variance itself is needed